jdaviz/app.py

```diff
@@ -1347,7 +1393,7 @@ def _update_selected_data_items(self, viewer_id, selected_items):
             if data.label not in active_data_labels:
                 viewer.remove_data(data)
                 viewer._layers_with_defaults_applied= [layer_info for layer_info in viewer._layers_with_defaults_applied  # noqa
-                                                       if layer_info['data_label'] != data.label]
+                                                       if layer_info['data_label'] != data.label]  # noqa
```

Comment:
There wasn't a codestyle fail here right? Is there a reason why a `# noqa` is required now?

Reply:
No reason, I think. Doesn't hurt but just confusing.